fun/lms/urls.py

```diff
@@ -54,13 +53,16 @@
     (r'^', include('fun_certificates.urls')),
 
 
-    # Override edx-platform urls
+    # Override edx-platform hard coded marketing urls
+    # See edx-platform/lms/urls.py:165
     url(r'^blog$', handler404),
     url(r'^donate$', handler404),
     url(r'^faq$', handler404),
     url(r'^jobs$', handler404),
     url(r'^press$', handler404),
     url(r'^media-kit$', handler404),
+    url(r'^contact$', RedirectView.as_view(url='//fun-mooc.help')),
```

Comment:
Why not enforcing HTTPS?
```suggestion
    url(r'^contact$', RedirectView.as_view(url='https://fun-mooc.help')),
```